### DISCUSSION TITLE
SMTP configuration [Aug 01]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3046] [Discord Threads]
> 🙋‍♂️ **NicolaiVdS**  
## hey i just created a form and when i wanne enable email at ... i get following message  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1002223588516040755/unknown.png)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Visit `Team & Settings`, `AppStore` & configure `SMTP`  
#  
> 👨‍🦲 **dstala**  
#### Only after configuring SMTP, you will be able to enable that option in forms  
#  
> 🙋‍♂️ **NicolaiVdS**  
#### <@857998597474484295> i have done that and still gives me that error  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1002223589220700251)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
